<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1304</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>591</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0.0227727, y1:0.472, x2:0.977273, y2:0.476955, stop:0 rgba(23, 98, 255, 255), stop:1 rgba(255, 150, 207, 255));
	font: 75 16pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
border-radius: 20
}
QPushButton:hover{
background-color:qlineargradient(spread:pad, x1:0.198909, y1:0.302, x2:0.892046, y2:0.687, stop:0 rgba(23, 98, 255, 255), stop:1 rgba(255, 150, 207, 255));
padding-bottom: 7;

}
QPushButton:pressed{
padding-top: 7;
padding-left: 7;
}
QPushButton:disabled{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(119, 119, 119, 255), stop:1 rgba(221, 221, 221, 255));
	color: rgba(241, 241, 241, 150);
}</string>
    </property>
    <property name="text">
     <string>识别</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/logo.png</normaloff>images/logo.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>591</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px solid;
font: 12pt &quot;微软雅黑&quot;;
border-radius:15;
border-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 250);
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>50</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0.0227727, y1:0.472, x2:0.977273, y2:0.476955, stop:0 rgba(23, 98, 255, 255), stop:1 rgba(255, 150, 207, 255));
	font: 40 12pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
border-radius: 20
}
QPushButton:hover{
background-color:qlineargradient(spread:pad, x1:0.198909, y1:0.302, x2:0.892046, y2:0.687, stop:0 rgba(23, 98, 255, 255), stop:1 rgba(255, 150, 207, 255));
padding-bottom: 7;

}
QPushButton:pressed{
padding-top: 7;
padding-left: 7;
}
QPushButton:disabled{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(119, 119, 119, 255), stop:1 rgba(221, 221, 221, 255));
	color: rgba(241, 241, 241, 150);
}</string>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>20</y>
      <width>1111</width>
      <height>801</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/images/bg.png);
border-radius:50</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>600</y>
      <width>261</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{
padding-bottom: 7;
background-color: rgba(255, 255, 255, 0);
}
QPushButton{
background-color: rgba(255, 255, 255, 0);
	color: rgb(0, 170, 255);
	font: 12pt &quot;微软雅黑&quot;;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/logo.png</normaloff>images/logo.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>220</y>
      <width>411</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px solid;
font: 12pt &quot;微软雅黑&quot;;
color: rgb(255, 0, 0);
border-radius:15;
border-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 250);
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>20</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 20pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>车辆信息记录</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>140</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>20</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 20pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>停车场信息发布</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>171</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>130</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(118, 118, 118);
font: 11pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>当前剩余车位</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>label_3</zorder>
   <zorder>pushButton</zorder>
   <zorder>textBrowser</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>textBrowser_2</zorder>
   <zorder>label_2</zorder>
   <zorder>label_4</zorder>
   <zorder>spinBox</zorder>
   <zorder>label_5</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1304</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resouce.qrc"/>
 </resources>
 <connections/>
</ui>
